cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -249,7 +256,7 @@ private void setNativeMessageView(JSONObject msgJson) throws ConsentLibException
         nativeView.setAttributes(new NativeMessageAttrs(msgJson));
     }
 
-    public void onMsgAccepted(Integer choiceId) {
+    public void onMsgAccepted(Integer choiceId) throws ConsentLibException{
```

Comment:
We should not change the signature of any public methods, here we can catch the exception instead

Reply:
will do this.